x-pack/plugins/security_solution/public/detection_engine/rule_management/api/hooks/use_bulk_action_mutation.ts

```diff
@@ -57,10 +57,8 @@ export const useBulkActionMutation = (
           if (updatedRules) {
             // We have a list of updated rules, no need to invalidate all
             updateRulesCache(updatedRules);
-          } else {
-            // We failed to receive the list of update rules, invalidate all
-            invalidateFindRulesQuery();
           }
+          invalidateFindRulesQuery({ refetchType: 'none' });
```

Comment:
It introduces a high probability that we will display stale data eventually. Refetch type `none` prevents hooks that are currently rendered to refetch their data. Therefore if there is no `updatedRules` in the response, they will continue to display outdated data.